app/models/reports/trainee_report.rb

```diff
@@ -391,6 +389,14 @@ def qts_or_eyts
       trainee.award_type
     end
 
+    def withdrawal_trigger
+      trainee.current_withdrawal&.trigger
+    end
+
+    def withdrawal_future_interest
+      trainee.current_withdrawal&.future_interest
+    end
+
     def withdraw_reasons
```

Comment:
Same question on the other PR #5058  (now I'm wondering if I'm forgetting some decisions during my time off) should this not be pulling from the new relationship `trainee.trainee_withdrawals.last.reasons`?

Reply:
Totally right. Had a brain lapse there.